.github/workflows/release.v1.yaml

```diff
@@ -0,0 +1,17 @@
+name: Release gcfg.v1
+on:
+  push:
+    branches:
+      - v1
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: tatskaari/release-action@master
+        with:
+          release-files: .
```

Comment:
Can you just get rid of this? If not, I should fix that.

Reply:
I can't tell. Any way of testing this before merge?